internal/backend/runtime/omni/controllers/omni/internal/talos/schematic.go

```diff
@@ -83,6 +91,18 @@ func GetSchematicInfo(ctx context.Context, c *client.Client, defaultKernelArgs [
 		return SchematicInfo{}, err
 	}
 
+	if inAgentMode {
+		id, idErr := pointer.To(schematic.Schematic{}).ID()
+		if idErr != nil {
+			return SchematicInfo{}, fmt.Errorf("failed to calculate extensions schematic ID: %w", idErr)
+		}
+
+		return SchematicInfo{
```

Comment:
can't we just return an empty schematic here?

Reply:
We need to return the ID of an empty schematic, which we do here. Not an empty string.